README.md

```diff
@@ -1,3 +1,18 @@
 # Literature Reviews
 
 The goal of this repo is to house markdown documents exploring proposed language and library additions to the Elm language.
+
+
+## Gathering feedback before submitting
+
+It's often helpful to get feedback on your concept before getting into the detail required for a literature review. You may open an issue on this repo to start a high-level discussion, with the goal of eventually creating a literature review pull request.
+
+## The process
+
+* Fork the https://github.com/elm-community/literature-reviews
+* Copy TEMPLATE.md to text/my-feature-name.md
```

Comment:
Why not `my-feature-name/README.md`? That way, example code and supporting files could live next to the review and GitHub would show a nice overview on the web.